user-service/src/main/resources/application-redis.yaml

```diff
@@ -0,0 +1,30 @@
+# Copyright 2020 Crown Copyright
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+spring:
+  autoconfigure:
+    exclude: []
+  cache:
+    type: redis
+    redis:
+      timeToLive: 60m
+  # redis.host/port or redis.cluster.nodes configured by k8s
+
+# TODO: spring-boot 2.2.x has cluster bugs, but 2.3.x has health-check bugs
+# Once released, if spring-boot 2.4.x fixes the health-check bugs, re-enable the endpoint here
+management:
+  health:
+    redis:
+      enabled: false

```

Comment:
Should we add this to the backlog?

Reply:
@dev930018 whats the expected release date for Spring 2.4? is it in development/predicted?

Reply:
https://spring.io/blog/2020/06/29/spring-boot-2-4-0-m1-is-now-available
October-November?

Reply:
yes, add a ticket on the backlog

Reply:
Raised PAL-734 with due-date October